drivers/overlay/check_116.go

```diff
@@ -0,0 +1,26 @@
+// +build go1.16
```

Comment:
What happens when go.1.17 happens?

Reply:
Each version of Go continues to define the build tags for versions that preceded it.